src/main/java/com/google/api/codegen/discovery/config/java/JavaTypeNameGenerator.java

```diff
@@ -61,12 +62,23 @@ public String getSubpackage(boolean isRequest) {
   }
 
   @Override
-  public String formatValue(String value, Field.Kind kind) {
-    switch (kind) {
-      case TYPE_STRING:
-        return "\"" + value + "\"";
-      default:
-        throw new IllegalArgumentException("unsupported kind: " + kind.toString());
+  public String getExample(String format) {
```

Comment:
Seems to be in the spirit of MVVM that only the content of these special-case comments should be in language-specific code ... would you agree?